src/server/models/url.ts

```diff
@@ -102,11 +107,6 @@ export const Url = <UrlTypeStatic>sequelize.define(
       type: Sequelize.BOOLEAN,
       allowNull: false,
     },
-    isSearchable: {
-      type: Sequelize.BOOLEAN,
-      allowNull: false,
```

Comment:
Do we need a database migration script to temporarily allow nulls in the table before the application is safe to deploy?

Reply:
Yea, working on it